azure/azure_build.yaml

```diff
@@ -154,6 +154,8 @@ jobs:
           sudo docker exec app /bin/bash -c ". ${{ variables.BASHRC }} && cd ${{ variables.DOCKER_WORKING_DIR }} && \
           lcov --capture --directory . --output-file coverage.info && \
           lcov --remove coverage.info '/usr/*' --output-file coverage.info && \
+          lcov --remove coverage.info '*/numpy/*' --output-file coverage.info && \
+          lcov --remove coverage.info '/home/mdolabuser/.pyenv/versions' --output-file coverage.info && \
```

Comment:
Actually, in order to make this more general, maybe we should put `*/.pyenv/versions` instead?